tests/integration/cloudformation/test_cloudformation_kinesis.py

```diff
@@ -24,3 +37,10 @@ def test_stream_creation(kinesis_client, deploy_cfn_template, snapshot):
 
     outputs = deploy_cfn_template(template=template).outputs
     snapshot.match("stack_output", outputs)
+
+    if not is_aws_cloud():
+        time.sleep(1)
```

Comment:
this also is something that we probably should dig into a bit at some point. Seems the stack deploys "successfully" but the kinesis stream isn't actually running yet, right?

Reply:
yes, to me it seems that AWS sets the status Complete for the stack when the stream is ready.